src/components/GeneralInfo/GeneralInformation/GeneralInformation.js

```diff
@@ -60,7 +60,7 @@ class GeneralInformation extends Component {
     };
 
     componentDidMount() {
-        this.props.loadSystemDetail && this.props.loadSystemDetail(this.props.entity.id);
+        this.props.loadSystemDetail && this.props.loadSystemDetail(this.props.inventoryId);
```

Comment:
I'd change this so we don't make the `inventoryId` required yet (edge for instance doesn't pass it yet)
```suggestion
        this.props.loadSystemDetail?.(this.props.inventoryId || this.props.entity.id);
```

Reply:
Aha, I will do so